lib/oxidized/model/riverbed.rb

```diff
@@ -0,0 +1,131 @@
+class Riverbed < Oxidized::Model
+  using Refinements
+
+  # Define the prompt
+  PROMPT = /^.*\s*[\w-]+\s*[#>]\s*$/i
+  prompt PROMPT
+
+  # Define comment character
+  comment '! '
+
+  # Use procs to remove unwanted lines
+  procs do
+    # Remove "Last login" message
+    proc { |output| output.gsub!(/^Last login:.*\n/, '') }
+  end
+
+  # Remove sensitive information
+  cmd :secret do |cfg|
+    cfg.gsub! /^(\s*tacacs-server (.+ )?key) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*username .+ (password|secret) \d) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*ntp server .+ key) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*ntp peer .+ key) .+/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*snmp-server community).*/, '\\1 <configuration removed>'
+    cfg.gsub! /^(\s*ip security shared secret).*/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*service shared-secret secret client).*/, '\\1 <secret hidden>'
+    cfg.gsub! /^(\s*service shared-secret secret server).*/, '\\1 <secret hidden>'
+    cfg
+  end
+
+  # Get version information and store it
+  cmd 'show version' do |cfg|
+    # Remove the command echo and the hostname from the output
+    cfg = cfg.cut_head
+    cfg.gsub!(PROMPT, '')
+
+    # Extract relevant information
+    comments = []
+    cfg.each_line do |line|
+      line.strip!
+      comments << "Product name: #{Regexp.last_match(1)}" if line =~ /^Product name:\s+(.*)$/
+      comments << "Product release: #{Regexp.last_match(1)}" if line =~ /^Product release:\s+(.*)$/
+      comments << "Build ID: #{Regexp.last_match(1)}" if line =~ /^Build ID:\s+(.*)$/
+      comments << "Build date: #{Regexp.last_match(1)}" if line =~ /^Build date:\s+(.*)$/
+      comments << "Build arch: #{Regexp.last_match(1)}" if line =~ /^Build arch:\s+(.*)$/
+      comments << "Built by: #{Regexp.last_match(1)}" if line =~ /^Built by:\s+(.*)$/
+      comments << "Product model: #{Regexp.last_match(1)}" if line =~ /^Product model:\s+(.*)$/
+      comments << "Number of CPUs: #{Regexp.last_match(1)}" if line =~ /^Number of CPUs:\s+(.*)$/
+    end
+    @version_info = comments.join("\n")
+    ''
+  end
+
+  # Get hardware information and store it
+  cmd 'show hardware all' do |cfg|
+    # Remove the command echo and the hostname from the output
+    cfg = cfg.cut_head
+    cfg.gsub!(PROMPT, '')
```

Comment:
Consider using [cut_both](https://github.com/ytti/oxidized/blob/master/docs/Ruby-API.md#cut_both)

Reply:
Replaced with cfg = cfg.cut_both to remove both the first and last lines in one step.